# config.yaml

inference_controller:
  ros__parameters:
    joint_state_topic: "/joint_states"
    joint_target_pos_topic: "/PD_control/command"
    rate: 10  # not used
    simulation: True
